src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { color, name } = props;
+
+  return (
+    <div className="button" style={{ backgroundColor: color }}>
+      <h2>
+        {name}
+      </h2>
+    </div>
+  );
+};
+
+
+Button.propTypes = {
+
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
```

Comment:
You're really moving fast which is nice 👍 
I guess `color` would be a `boolean` in later milestone if thou it not required in this milestone.